global.d.ts

```diff
@@ -0,0 +1,153 @@
+import {
+  Component, ComponentType, ComponentClass, CSSProperties, ReactElement, Ref
+} from 'react'
+
+declare global {
+  // @see https://stackoverflow.com/questions/38383676/error-ts2339-property-connection-does-not-exist-on-type-navigator
+  interface Navigator extends NavigatorNetworkInformation {}
```

Comment:
I'm not sure about overriding built-in interfaces in npm packages. This is a normal approach in the "final" project, but in npm package this is questionable